crates/tasks_ui/src/modal.rs

```diff
@@ -39,12 +39,10 @@ impl Spawn {
 /// Rerun last task
 #[derive(PartialEq, Clone, Deserialize, Default)]
 pub struct Rerun {
-    /// Controls whether the task context is reevaluated prior to execution of a task.
-    /// If it is not, environment variables such as ZED_COLUMN, ZED_FILE are gonna be the same as in the last execution of a task
-    /// If it is, these variables will be updated to reflect current state of editor at the time task::Rerun is executed.
-    /// default: false
+    /// Overrides `reevaluate_context` property of the task being reran.
```

Comment:
Interesting, I would love to see the test on this as it's new logic we add here.